Basic/03_variable/string_variable.py

```diff
@@ -2,28 +2,35 @@
 # sebagai serangkaian karakter yang berdekatan
 # yang diwakili dalam tanda kutip.
 
+# Dalam membuat variabel string kita dapat
+# menggunakan petik dua ("") / petik satu ('')
 pesan = "saya pergi ke pasar"
+pesan = 'saya pergi ke pasar'
```

Comment:
string pada python disarankan menggunakan double quote
```suggestion
pesan = "saya pergi ke pasar"
```